bin/node/testing/src/genesis.rs

```diff
@@ -52,6 +52,9 @@ pub fn config_endowed(code: Option<&[u8]>, extra_endowed: Vec<AccountId>) -> Run
 			code: code.map(|x| x.to_vec()).unwrap_or_else(|| wasm_binary_unwrap().to_vec()),
 			..Default::default()
 		},
+		identity: IdentityConfig {
+			identities: vec![(alice(), "Alice".to_string()), (bob(), "Bob".to_string())],
```

Comment:
Be mindful that `Data` in `IdentityInfo` is truncated to 32 bytes (see docs), so it is recommended to pass a hash of the data rather than the data itself for stuff like `display`.